webpack/assets/javascripts/react_app/ReactApp/ReactApp.js

```diff
@@ -2,14 +2,13 @@ import React from 'react';
 import PropTypes from 'prop-types';
 import { Router } from 'react-router-dom';
 import history from '../history';
-
 import Layout from '../components/Layout';
 import AppSwitcher from '../routes';
 
-const ReactApp = ({ data: { layout } }) => (
+const ReactApp = ({ data: { layout, metadata } }) => (
```

Comment:
we decided to avoid this prop drilling, [metadata is in the redux state](https://github.com/theforeman/foreman/blob/develop/webpack/assets/javascripts/react_app/ReactApp/index.js#L25) so you should use selectors.